docs/metrics.md

```diff
@@ -26,37 +26,36 @@ HTTP code.
 
 ## Routes
 
-Routes are organized in the following fashion:
-`/_/metrics/<extension-type>/<location-name/[<metric-type>]/[<bucket>]/[<key>]?[versionId=<version-id>]`
+Routes are organized as follows:
+`/_/backbeat/api/<extension-type>/<location-name/[<metric-type>]/[<bucket>]/[<key>]?[versionId=<version-id>]`
 
 Where:
 
-- `<extension-type>` can currently only support `crr` for replication metrics
+- `<extension-type>` currently supports only `crr` for replication metrics
 - `<location-name>` represents any current destination replication locations you
-  have defined. If you would like metrics displayed for all locations, use `all`
-- `<metric-type>` is an optional field. Leaving this out will get all metrics
-  available for given extension and location. If you specify a metric type, you will
-  get the metric specified.
-- `<bucket>` is an optional field. If provided, it is the name of the bucket in
-  which the object exists.
-- `<key>` is an optional field. If provided, it is the object's key when getting
-  CRR metrics for a particular object.
-- `<version-id>` is an optional field. If provided, it is the object's version
-  ID when getting CRR metrics for a particular object.
+   have defined. To display metrics for all locations, use `all`
+- `<metric-type>` is an optional field. If you specify a metric type, Backbeat
+   returns the specified metric. If you omit it, Backbeat returns all available
+   metrics for the given extension and location. 
+- `<bucket>` is an optional field. It carries the name of the bucket in which
+   the object is expected to exist.
+- `<key>` is an optional field. When getting CRR metrics for a particular object,
+   it contains the object's key. 
+- `<version-id>` is an optional field. When getting CRR metrics for a particular
+   object, it contains the object's version ID.
 
-### `/_/metrics/crr/<location-name>`
+### `/_/backbeat/api/<location-name>`
```

Comment:
Add `/metrics/crr` between `api` and `/<location-name>` (same for the following occurrences).